packages/@o3r/localization/schemas/localization.schema.json

```diff
@@ -0,0 +1,83 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "LocalizationSchema",
+  "description": "Schema of Localization file",
+  "type": "object",
+  "additionalProperties": false,
+  "patternProperties": {
+    "\\$schema": true,
+    "^[a-zA-Z0-9.-]+": {
+      "type": "object",
+      "oneOf": [
```

Comment:
shouldn't we have a oneOf for ref / dictionary / basic ?
because for basic I would expect to have `defaultValue` required

Reply:
There you have set defaultValue mandatory even when dictionary is true
I would expect something like 
```json5
{
...
  "oneOf": [
    { required: ["description", "defaultValue"], properties: { ..., dictionary: { value: false } }, ... },
    { required: ["description", "dictionary"], properties: { ..., dictionary: { value: true } }, ... },
    { required: ["description", "$ref"], properties: { ..., dictionary: { value: false } }, ... },
  ]
}
```
maybe I'm mistaken

Reply:
you're right, I did the fix quickly between 2 calls and forgot it.
I will add a 3rd case.